TSP/TestsServicesTSP.Moq/EmployeeServiceTests.cs

```diff
@@ -0,0 +1,109 @@
+using AutoMapper;
+using BLL.AutoMapper;
+using BLL.Models;
+using BLL.Services;
+using DAL.Entities;
+using DAL.Interfaces;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace TestsServicesTSP.Moq
```

Comment:
Should be named as a project which you test, ex. "{ProjectName.UnitTests}"

Reply:
fix